qdrant-landing/content/documentation/concepts/collections.md

```diff
@@ -116,6 +116,27 @@ await client.CreateCollectionAsync(
 );
 ```
 
+```go
+import (
+	"context"
+
+	"github.com/qdrant/go-client/qdrant"
+)
+
+client, err := qdrant.NewClient(&qdrant.Config{
+	Host: "localhost",
+	Port: 6334,
+})
+
+client.CreateCollection(context.Background(), &qdrant.CreateCollection{
```

Comment:
We don't use the return values since these snippets mainly intend to demonstrate the method and property names in the SDKs.